app/scripts/abos/overview/abosoverview.controller.js

```diff
@@ -22,6 +22,7 @@ angular.module('openolitor-admin')
       $scope.loading = false;
       $scope.selectedAbo = undefined;
       $scope.model = {};
+      $scope.vertriebL = [];
```

Comment:
i would rename this variable to vertriebList as vertriebL and vertrieb are quite close